ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh

```diff
@@ -6,8 +6,8 @@ set -o pipefail
 
 set +x
 
-PACKET_PROJECT_ID=$(cat "${CLUSTER_PROFILE_DIR}/packet-project-id")
-PACKET_AUTH_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/packet-auth-token")
+PACKET_PROJECT_ID=$(cat "/var/run/secrets/packet/${PACKET_ACCOUNT}/packet-project-id")
+PACKET_AUTH_TOKEN=$(cat "/var/run/secrets/packet/${PACKET_ACCOUNT}/packet-auth-token")
```

Comment:
As per the previous comment, this could become something like as:
```
PACKET_PROJECT_ID=$(cat "${CLUSTER_PROFILE_DIR}/${PACKET_ACCOUNT}-packet-project-id")
PACKET_AUTH_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/${PACKET_ACCOUNT}-packet-auth-token")
```